examples/custom_contols/simulation.toml

```diff
@@ -93,7 +93,7 @@ enabled = false
 
 [Profiles]
 "Use profile manager" = true
-source_type = "HDF5"
+source_type = "h5"
```

Comment:
I changed this to match the FileFormat enum that is now defined. This forced the rename of the file PyDSS/ProfileManager/hooks/h5.py.